src/univariate/continuous/ksdist.jl

```diff
@@ -120,16 +120,16 @@ function ceil_rems_mult(n,x)
 end
 
 # n!*(e/n)^n
-function stirling(n)    
+function stirling(n)
     if n < 500
-        s = 1.0
+        s = 1
```

Comment:
I can take a look, but that seems like it might be out of scope for this PR.

I do appreciate the very thorough review, though. I will get all these small things fixed soon.